Evolve.Migrations.Helper/Program.cs

```diff
@@ -83,8 +86,8 @@ private static string GetRootPath(IReadOnlyList<string> args)
                     rootPath = "db/migrations";
                     break;
                 default:
-                    Console.WriteLine($"{Assembly.GetExecutingAssembly().GetName().Name}: '{command}' is not a valid command.\r\n");           
-                    PrintHelpMessage();
+                    Console.WriteLine($"{Assembly.GetExecutingAssembly().GetName().Name}: '{command}' is not a valid command.\n");           
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/odair-pedro/evolve-migrations-helper/pullRequest?prid=5663793)